src/main/java/org/terasology/metalrenegades/world/dynamic/DynamicWorldGenerator.java

```diff
@@ -67,6 +68,7 @@ protected WorldBuilder createWorld() {
                 .addRasterizer(new TreeRasterizer())
                 .addRasterizer(new OreRasterizer())
                 .addRasterizer(new DiscoverablesRasterizer())
+                .addRasterizer(new SunlightRasterizer(-20))
```

Comment:
This is need to have the LOD chunks properly lit. This is what it looks without this additional rasterizer:

![Terasology-210813190644-1920x1055](https://user-images.githubusercontent.com/1448874/129398411-801caef2-97f3-466e-81e9-df732a8e968e.png)